### DISCUSSION TITLE
Has the Resolution Handling Code Been Pushed Upstream?

### DISCUSSION THREAD

**Discussion by 1457384613gh** [dakanji/RefindPlus#145] [General]
As for [codes around gop to support more resolutions to set as same as the screen], have you pulled requests to rEFInd????
[codes around gop to support more resolutions to set as same as the screen] are very great!
Your codes can prevent icons from being stretched horizontally, when using old GPU around Ironlake.
Will rEFInd merge [codes around gop to support more resolutions to set as same as the screen]????
Will rEFInd support more resolutions like RefindPlus, in the future????? 
Some users, not me, are highly demanding.
They would like both [secure boot] and [resolution of bootloader as same as that of screen].

**Comment by dakanji** [comment]
Not likely there will be a pull request on these items as focus is on improving RefindPlus.
Anyone can of course package things and make a pull request to that project
Queries on what rEFInd might do should be sent to that project
